kernel/kata_config_version

```diff
@@ -1 +1 @@
-55
+56
```

Comment:
This sort of change should not require a bump in the config version. We should fix this in a future PR. We should enforce a version bump only if any of the kernel configuration changes imho.